### PR TITLE
WIP: Relicense the port library to EPLv2 & Apache v2

### PR DESCRIPTION
**Pull request by DanHeidinga**
Update the license as per the outcome of the vote in issue #1596.

This can't be accepted until all contributors in  #1606 have signed off on the relicensing their contributions.

### CODE REVIEW

**Comment by DanHeidinga** [comment]
@mstoodle Can you confirm this is the correct license text?  In particular, not sure about the gpl / class path part

**Comment by charliegracie** [comment]
I looked at a few files and the text appears to be correct.  Thanks for jumping on this work but note we have not given the ok to make the switch yet so it may be a while before we can merge these changes.

File: port/zos390/omrvmem_support_below_bar_64.s [hunk 1]
```diff
@@ -204,30 +204,6 @@ OMRSTORAGE RMODE ANY                                                     000000
 @@CCN@19 ALIAS C'omrfree_memory_below_bar'                               000000
 @@CCN@11 ALIAS C'omrallocate_4K_pages_below_bar'                         000000
 @@CCN@1  ALIAS C'omrallocate_1M_pageable_pages_below_bar'                000000
-************************************************************************
```
Comment (charliegracie) [inline port/zos390/omrvmem_support_below_bar_64.s]: I believe the duplicate copy was there so that the generated file also contained the copyright

Comment (DanHeidinga) [reply port/zos390/omrvmem_support_below_bar_64.s]: That's annoying.  I'll check on that and update here once I've confirmed

Comment (DanHeidinga) [reply port/zos390/omrvmem_support_below_bar_64.s]: Compiled on zOS and only see these files:
```
$ cd ~/omr/port
$ find . -name "omrvmem_support_*"                                    
./omrvmem_support_above_bar.o
./omrvmem_support_below_bar_64.o
./zos390/omrvmem_support_below_bar_64.s
./zos390/omrvmem_support_below_bar_31.s
./zos390/omrvmem_support_above_bar.s
```

Is there a specific generated file that needs to be looked at?  I'm not able to find it...

**Comment by charliegracie** [comment]
This will need to be rebased.

There were last minute changes to the copyright header which will need to be incorporated into these changes.  You can see this change in #1606

**Comment by DanHeidinga** [comment]
Closing this in favour of https://github.com/eclipse/omr/pull/1622 which has the correct copyrights.

**Pull request closed without merging**